scripts/check-reqs.sh

```diff
@@ -2,17 +2,17 @@
 
 # Checks if the tools required for properly building, verifying and testing GlusterD are installed.
 
-TOOLS=(glide gometalinter)
+TOOLS=(glide gometalinter etcd)
 
 MISSING=0
 
 for tool in ${TOOLS[@]}; do
   type $tool >/dev/null 2>&1
   if [ $? -ne 0 ]; then
-    echo "$tool is missing on the system"
+    echo "$tool package is missing on the system"
```

Comment:
it should not be tool package. because suppose etcd is missing then value of $tool will be etcd, and according to your opinion if i change it to too/package then it will become  etcd/package missing.